ogr/ogrsf_frmts/generic/ogrlayerarrow.cpp

```diff
@@ -318,8 +321,44 @@ int OGRLayer::GetArrowSchema(struct ArrowArrayStream *,
                 break;
 
             case OFTDateTime:
-                psChild->format = "tsm:";
+            {
+                const char *pszPrefix = "tsm:";
+                const char *pszTZOverride =
+                    m_aosArrowArrayStreamOptions.FetchNameValueDef("TIMEZONE",
+                                                                   "");
```

Comment:
This option is only meant to be used internally (so that GPKG driver can set it)? 
(since I don't see it documented; it's probably also not really useful to expose)

Reply:
yes, this was internal only. But I've decided to expose it, if user want to force a timezone if they have knownledge the driver doesn't have